cypress/integration/AdminApproval_spec.js

```diff
@@ -25,14 +27,58 @@ describe('AdminApproval', () => {
       .get('.insurances').should('contain', 'Schinner, Kreiger and Stokes, Yundt-Cronin, Herman-Bogan')
       .get('.all-unvetted').should('have.length', 1)
   })
+
   it('Should display an approve and deny button for each card', () => {
     cy.get('.approve-button').should('be.visible')
       .get('.deny-button').should('be.visible')
   })
+
   it('Should display a message stating there are no unvetted suggestions if all providers are vetted', () => {
 
   })
+
   it('Should display an error message when there is an error thrown in the fetch request', () => {
 
   })
+
+  it('Should make a patch request when the user clicks the approve button', () => {
+    cy.fixture('mockVetted.json')
+      .then(modifiedData => {
+        cy.intercept('PATCH', 'https://head-to-toe-be.herokuapp.com/api/v1/medical_professionals/80', modifiedData)
+      })
+
+    cy.fixture('mockVetted.json')
```

Comment:
Great job with using 3 intercepts to test that it got approved and ended up in the right place!